### ISSUE TITLE
notes for session 4

### ISSUE THREAD

**Issue by konradpa** [lnnrtwttkhn/version-control-course-uhh-ws23#23]

**Comment by lnnrtwttkhn** [comment]
re permission denied issue:

```bash
git init /bla/banana
fatal: cannot mkdir /bla/banana: Read-only file system
```

and

```bash
git init /Users/konrad/la
fatal: cannot mkdir /Users/konrad/la: Permission denied
```
this is a path issue. recommendation

1. First `cd` into the directory where you want to create a repository
1. Use relative paths in `git init`

**Comment by lnnrtwttkhn** [comment]
we had one weird issue that we couldn't fully resolve, where the Git Bash of a Windows user was in one folder but this wasn't synchronous with the Filer Explorer in Windows. The difference was in the capitalization of the user name in the path `/Users/username` vs. `Users/Username` although the could not find the `Users/Username` in the File explorer.

**Comment by lnnrtwttkhn** [comment]
- [ ] explain difference in the order of git commit hashes when using `git diff`

**Comment by lnnrtwttkhn** [comment]
- [ ] show why word is not useful for git
- [ ] make reference to wildcard section in command line chapter
- [ ] emphasize why it's important to create a clean commit history
- [ ] explain what `HEAD -> main`  means and what `HEAD` is

**Comment by lnnrtwttkhn** [comment]
- [ ] CLI chapter: demonstrate `wget` command

**Comment by konradpa** [comment]
> I realized we should follow our recipes example more thoroughly, instead of working with `example.txt` files etc. I would prepare a "proper" recipes repository until next week.

Yes I think we should emphasize that after next session everyone should have an initialized repositories with a recipe  file(a md. file?). Maybe we should also give time to actually put a recipe in the file?
Especially important before we start with github
